doc/train/multi-task-training.md

```diff
@@ -53,9 +53,21 @@ Specifically, there are several parts that need to be modified:
     `"descriptor": "my_descriptor", `
     to replace the previous detailed parameters. Here, you can also specify the shared_level, such as
     `"descriptor": "my_descriptor:shared_level", `
-    and use the user-defined integer `shared_level` in the code to share the corresponding module to varying degrees
-    (default is to share all parameters, i.e., `shared_level`=0).
-    The parts that are exclusive to each model can be written following the previous definition.
+    and use the user-defined integer `shared_level` in the code to share the corresponding module to varying degrees.
+- - For descriptors, `shared_level` can be set as follows:
+    - Valid `shared_level` values are 0-1, depending on the descriptor type
+    - Each level enables different sharing behaviors:
+      - Level 0: Shares all parameters (default)
+      - Level 1: Shares type embedding only
+    - Not all descriptors support all levels (e.g., se_a only supports level 0)
+- - For fitting nets, we only support the default `shared_level`=0, where all parameters will be shared except for `bias_atom_e` and `case_embd`.
+- - To conduct multitask training, there are two typical approaches:
+    1. **Descriptor sharing only**: Share the descriptor with `shared_level`=0. See [here](../../examples/water_multi_task/pytorch_example/input_torch.json) for an example.
+    2. **Descriptor and fitting network sharing with data identification**:
+       - Share the descriptor and the fitting network with `shared_level`=0.
+       - {ref}`dim_case_embd <model[standard]/fitting_net[ener]/dim_case_embd>` must be set to the number of model branches, which will distinguish different data tasks using a one-hot embedding.
+       - See [here](../../examples/water_multi_task/pytorch_example/input_torch_sharefit.json) for an example.
+- - The parts that are exclusive to each model can be written following the previous definition.
```

Comment:
![image](https://github.com/user-attachments/assets/79e1b10a-1b3c-40ea-a4d9-4363da5b09a4)

What's the reason to add two `-`?